mirrord/protocol/proto/api.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package mirrord.api;
+
+message Empty {}
+
+message BincodeMessage {
+  bytes encoded = 1;
+}
```

Comment:
I somehow expected that the protocol messages would be defined as protocol buffer messages. If all we're sending are already unstructured bytes (as far as protocol buffers are concerned), why do we use protocol buffers at all?

Reply:
It's a good question and should be written somewhere in the draft for better discussion.
We wanted to have a way to convert functionality piece by piece, without needing to rewrite the whole agent. The solution @DmitryDodzin came up with is that the new protocol will contain the old protocol so we can start moving parts to the new protocol as we go.